src/models/community_business.ts

```diff
@@ -602,6 +603,27 @@ export const CommunityBusinesses: CommunityBusinessCollection & CustomMethods =
             } , {});
           return { age };
         }),
+      lastWeek: applyQueryModifiers(client('visit')
+        .select({ createdAt: 'visit.created_at' })
+        .whereRaw('visit.created_at >= CURRENT_DATE - INTERVAL \'7 day\'')
+        .innerJoin('visit_activity', 'visit_activity.visit_activity_id', 'visit.visit_activity_id')
+        .innerJoin('user_account', 'user_account.user_account_id', 'visit.user_account_id')
+        .innerJoin('gender', 'gender.gender_id', 'user_account.gender_id'), queryMatchOnColumnNames)
+        .then((data) => {
+          const lastWeekDatesObject = Array
```

Comment:
I don't think it's necessary to construct the object with zeros here if you just overwrite it below.